### PR TITLE
Change logic for non-editable systems

### PR DESCRIPTION
**Pull request by MartinSchuhmacher**
# Short Description
Change logic for non-editable systems inside of the new administration page

## Links to Ticket and related Pull-Requests
https://ticketsystem.hpi-schul-cloud.org/browse/BC-191

## Changes
- make systems non-editable that are zentral IServ or old authentication methods 
- make zentral IServ non-removable

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/components/organisms/administration/AuthSystems.vue [hunk 1]
```diff
@@ -108,7 +109,14 @@ export default {
 		...mapActions("schools", ["deleteSystem"]),
 		// TODO - Discuss which systems are still gonna be editable in the future
 		isEditable(system) {
-			return system.type === "ldap";
+			return system.type === "ldap"
+				? system.ldapConfig.provider === "general"
```
Comment (pkleybolte) [inline src/components/organisms/administration/AuthSystems.vue:113]: That looks like `system.type === "ldap" && system.ldapConfig.provider === "general"`

File: src/components/organisms/administration/AuthSystems.vue [hunk 2]
```diff
@@ -108,7 +109,14 @@ export default {
...
+				: false;
+		},
+		isRemovable(system) {
+			return system.type === "ldap"
```
Comment (pkleybolte) [inline src/components/organisms/administration/AuthSystems.vue:117]: That looks like `system.type !== "ldap" || system.ldapConfig.provider === "general"`

**Review by pkleybolte — CHANGES_REQUESTED** [review]
Is this behavior enforced in the server as well?

File: src/components/organisms/administration/AuthSystems.vue (hunk 1 above)
Comment (MartinSchuhmacher) [reply src/components/organisms/administration/AuthSystems.vue]: thank you, I changed that

File: src/components/organisms/administration/AuthSystems.vue (hunk 2 above)
Comment (MartinSchuhmacher) [reply src/components/organisms/administration/AuthSystems.vue]: that as well ?:-)

**Review by pkleybolte — APPROVED** [review]

**Pull request merged**